code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -53,48 +53,51 @@
 	robotize("Morpheus Cyberkinetics")
 	..()
 
-/obj/item/organ/cell
+/obj/item/organ/internal/cell
```

Comment:
Perhaps this organ could be used for powering cybernetic limbs and implants, and certain implants could act as a power source? So, for IPCs, they'd have an impetus to regularly recharge, while for organics they'd need to have something like a calorie-to-energy implant or some other power source - perhaps IPC APC charging could be implemented as an implant! Out-of-scope, though.